cmd/kubeadm/app/util/config/cluster.go

```diff
@@ -50,6 +51,8 @@ func loadConfigurationBytes(client clientset.Interface, w io.Writer, logPrefix,
 
 	fmt.Fprintf(w, "[%s] Reading configuration from the cluster...\n", logPrefix)
 
+	// TODO: This code should support reading the MasterConfiguration key as well for backwards-compat
+	// Also, the key really should be ClusterConfiguration...
```

Comment:
I'm fine with naming the ConfigMap `kubeadm-config`, but renaming the constant from `InitConfigurationConfigMap` to `ClusterConfigurationConfigMap`.
+1 for renaming the ConfigMapKey